src/python/grpcio/grpc/_cython/_cygrpc/aio/rpc_error.pyx.pxi

```diff
@@ -14,7 +14,7 @@
 """Exceptions for the aio version of the RPC calls."""
 
 
-cdef class _AioRpcError(Exception):
+cdef class AioRpcError(Exception):
```

Comment:
`AioRpcError` it's an internal `Exception` only used between the gRPC cython and gRPC Python world, the developer knows nothing since the `cygrpc. AioRpcError` is cast to a `grpc.aio.experimental.AioRpcError` during the `Call.__awaIt__` which is a Python class that derivates from the `grpc.RpcError`.